vtex/faststore
# chore: renames from first to numberOfItems in Shelves

## What's the purpose of this pull request?

This PR aims to rename the `first` prop in the `ProductShelf` and `CrossSellingShelf` since the CMS.

## How to test it?

1. in faststore.config.js change the workspace to `formspace`
2. `yarn dev` to run locally.
3. check the Cross Selling Shelves in pdp  or `ProductShelf` in homepage using these links (version from `formspace` workspace):

pdp
http://localhost:3000/api/preview?contentType=pdp&documentId=717bc97f-8f19-4744-a52b-7ccac43cc898&versionId=01b9e4f3-4253-42ee-a8a3-96c6bdea25fa

homepage
http://localhost:3000/api/preview?contentType=home&documentId=ad2fd81d-a53c-4281-8d01-a4fc2f274db3&versionId=97a6e029-47a2-4a6d-8276-e2266690cf04


### Starters Deploy Preview

TBD after this PR got merged (depends on cms sync in master workspace after the merge)
